ResXTranslator/TranslationServices/GoogleTranslation.cs

```diff
@@ -52,9 +55,10 @@ public async Task<string> TranslateAsync(string text, string language, string so
                     Console.WriteLine("The language code you entered could not be handled.");
                     Environment.Exit(-1);
                 }
-                var result = await _client.TranslateTextAsync(text, langCode.First(), sourceLanguage);
+                var result = _client.TranslateText(text, langCode.First(), sourceLanguage);
                 Console.WriteLine($"Translated {text} from {result.DetectedSourceLanguage} to {langCode.First()}");
                 translation = result.TranslatedText;
+                Thread.Sleep(1000);
```

Comment:
Is there a reason why you choosed to sleep the thread here instead of awaiting ?

Reply:
The issue was that Main was finishing before the task finished. I have added some await code to ensure we are waiting for Run (and all of its tasks) to finish before Main is also finished.